search_profiling/README.md

```diff
@@ -0,0 +1,13 @@
+# Search query profiling
+
+This provides a script which can run an elasticsearch query many times, using different search terms each time to overcome caching, and return statistics on the performance of the queries.
+
+## Usage
+
+1. Run `yarn` to install dependencies.
+2. Run `yarn refresh-templates` to get template files from the prod/stage search-templates.json endpoints.
```

Comment:
What about where we want to test custom queries not from `search-templates.json`? Might be useful to have an example custom query in here too.

Reply:
Looks like you included an example below (aggregations) - just a mention that you can do that perhaps.

Reply:
We do this in rank. Ping @harrisonpim - any thoughts on if we should bring profiling into rank?

https://github.com/wellcomecollection/catalogue-api/pull/122/files#diff-3695c3f2fb64cf49101811836b159b2672287c38969db5f143e0cfadb3c39600

Reply:
@kenoir this gets the files with the intention that they can be modified - I can probably improve the ergonomics of this a bit so that you can have more than one query file per index

Reply:
i feel like this is exactly the kind of thing we're working towards with rank, so yeah, would be great to join them up in the near future